tools/make_patch.c

```diff
@@ -7,6 +7,14 @@
 
 #define HIGH(x) (((x) >> 8) & 0xff)
 #define LOW(x) ((x) & 0xff)
+#define ROMBANKSTART 0x4000
+#define ROMEND 0x8000
```

Comment:
So, usually I'd agree these are a good idea, but after getting rid of the `offset` and `abs_offset` functions, I think just having the comments in `get_address_type_limit` with the numeric values right there is fine.

Reply:
Sounds good!